btrack/_localization.py

```diff
@@ -134,6 +169,11 @@ def segmentation_to_objects(
         If an intensity image has been provided, default to calculating the
         weighted centroid. See skimage.measure.regionprops for more info.
 
+    assign_class_ID : bool, default False
+        If specified, assign a class label for each individual object based on
+        the pixel intensity found in the mask. Requires semantic segmentation,
+        i.e. object type 1 will have pixel value 1. Assumes there is more than
```

Comment:
Can you finish this docstring

Reply:
yeah no need to finish that as i removed the check that excluded masks with only one instance in them